src/Features/CSharp/Portable/SignatureHelp/ConstructorInitializerSignatureHelpProvider.cs

```diff
@@ -82,8 +82,11 @@ private static bool IsArgumentListToken(ConstructorInitializerSyntax expression,
                 return null;
             }
 
+            var currentCunstructor = semanticModel.GetDeclaredSymbol(( constructorInitializer.Parent as ConstructorDeclarationSyntax ));
+
             var symbolDisplayService = document.Project.LanguageServices.GetService<ISymbolDisplayService>();
             var accessibleConstructors = type.InstanceConstructors
+                                             .WhereAsArray(c => c != currentCunstructor)
```

Comment:
:bulb: I would prefer this condition be merged with the next in order to avoid creating this array unnecessarily